database/nakadi/10_data/04_tables/05_timeline.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE zn_data.timeline (
   tl_switched_at     TIMESTAMP WITH TIME ZONE DEFAULT NULL,
   tl_cleanup_at      TIMESTAMP WITH TIME ZONE DEFAULT NULL,
   tl_latest_position JSONB                    DEFAULT NULL,
+  tl_deleted         BOOLEAN                  DEFAULT FALSE,
```

Comment:
Well, right now when event-type is deleted - it is removed from DB, so it will make no sense to keep timelines for ET that was deleted. 
But when ET still exists I think it's better to keep all timelines for the history (at least for now when timeline feature is not really tested well)